src/RNCloudStorage.ts

```diff
@@ -92,6 +92,16 @@ const RNCloudStorage = {
     return nativeInstance.readFile(verifyLeadingSlash(path), scope ?? defaultScope);
   },
 
+  /**
+   * Trigger a download of the file at the given path.
```

Comment:
I think it'd make sense to clarify that this function only effects iCloud here. Something like:

> Triggers the download of the file at the given path from iCloud. Does not have any effect on Google Drive.